packages/ui/src/forum/modals/CreateThreadModal/utils.ts

```diff
@@ -0,0 +1,5 @@
+import BN from 'bn.js'
+
+export const getMessage = (fee?: BN) => {
+  return `Insufficient funds to cover the thread creation. You need at least ${fee?.toString()} JOY on your account for this action.`
```

Comment:
Do we have some JOY formatting method? Not sure if `BN.toString()` would be consistent with currency formatting in other places.

Reply:
That's how it's done in warnings in other parts of the site, so I'm inclined to leave it as is for now.